projects/assets/src/translations/en/common.ts

```diff
@@ -142,6 +142,7 @@ export const common = {
     voucherExceeded:
       'This coupon has exceeded the number of times it can be used',
     unknownIdentifier: 'Item not found.',
+    invalidCostCenter: 'Invalid cost center.',
```

Comment:
Should probably move to the b2b checkout feature.

Reply:
To clarify this, I think plabadie is talking about `feature-libs/checkout/b2b/assets/translations/en/checkout-b2b.i18n.ts`